Vagrantfile

```diff
@@ -25,6 +30,11 @@ Vagrant.configure("2") do |config|
     raster_vision.vm.synced_folder "~/.aws", "/home/vagrant/.aws"
     raster_vision.vm.synced_folder DATA_DIR, "/opt/data"
 
+    # If RASTER_VISION_NOTEBOOK_DIR is set, sync it to the VM
+    if File.directory?(File.expand_path(NOTEBOOK_DIR))
```

Comment:
If notebook dir isn't set, which is only considered a warning, then this line will fail since it will try to call `expand_path` on nil. This line should be changed to 
```
    if NOTEBOOK_DIR and File.directory?(File.expand_path(NOTEBOOK_DIR))
``` 
to avoid this.